frontend/src/components/general/MultiLevelSelector.js

```diff
@@ -1,13 +1,21 @@
-import React from "react";
-import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
-
-import { List, ListItem, ListItemText, ListItemIcon, Typography, Divider } from "@material-ui/core";
+import {
+  Container,
+  Divider,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Typography,
```

Comment:
Nice! Not sure if it was intentional but I like the alphabetization. Helps with readability and thereby understandability IMO 🙏

Reply:
This is thanks to my Prettier extension in VSCode. Glad you like it! Eventually would we want something like this in our contributor style guide?

Reply:
JK, it's actually just VSCode editor settings

```
 "editor.codeActionsOnSave": {
        "source.fixAll": true,
        "source.organizeImports": true,
        "source.addMissingImports": true,
    },
```